.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,45 @@
+name: golangci-lint
+
+env:
+  SETUP_GO_VERSION: '^1.18'
+
+on:
+  schedule:
+    - cron:  '0 5 * * *'
+  pull_request:
+    paths-ignore:
+    - 'docs/**'
+    - 'charts/**'
+    - 'scripts/**'
```

Comment:
Are these default exceptions because most of the paths do not exist?

Reply:
Yes, copied those from fleet. I'd try to keep them similar, so I can update by copying in the future and don't have to diff.